packages/kbn-ci-stats-reporter/src/report_time.ts

```diff
@@ -19,7 +19,24 @@ export const getTimeReporter = (log: ToolingLog, group: string) => {
           group,
           id,
           ms: Date.now() - startTime,
-          meta,
+          meta: Object.fromEntries(
```

Comment:
What was the need to change this? Just an adhoc improvement or was something broken by adding peggy files to babel register?

Reply:
Just noticed that this was broken while working on this, I'll break it into it's own PR. Forgot it was there.